templates/provider/cluster-api-provider-openstack/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if and .Values.configSecret.create .Values.configSecret.name }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.configSecret.name }}
+  namespace: {{ .Values.configSecret.namespace | default .Release.Namespace | trunc 63 }}
+stringData:
+{{ toYaml .Values.config | indent 2 }}
```

Comment:
Could you please add an empty `config` to the `values.yaml` file and add schema for it?

Reply:
Done.